src/components/Editor/index.js

```diff
@@ -253,6 +253,13 @@ class Editor extends PureComponent<Props, State> {
     if (selectedSource && isLoaded(selectedSource.toJS())) {
       this.highlightLine();
     }
+
+    // NOTE: when devtools are opened, the editor is not set when the source loads
```

Comment:
The linter thinks this line is too long :)

Reply:
lol - i'll help it out